mixer/cmd/config.go

```diff
@@ -96,11 +98,49 @@ var configSetCmd = &cobra.Command{
 	},
 }
 
+var configGetCmd = &cobra.Command{
+	Use:   "get <property>",
+	Short: "Get a property in the config file",
+	Long: `This command will parse the provided property in the format 'Section.Property'
+	and return its value. The value returned is the value used by mixer, which will be
+	either the value set in the config file or the default value if the property has not
+	been defined in the file.`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		var value string
+
+		// Look for the value in mixer.state first
+		var ms config.MixState
+		err := ms.Load("")
+		if err == nil {
```

Comment:
Same comment as previous, are we sure we want to ignore not being able to load the mixer.state?